arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -0,0 +1,2929 @@
+            },
+            "virtualMachine": {
+              "$ref": "#/definitions/VirtualMachineConfiguration",
+              "description": "Virtualization-related configuration. Present only when `virtualizationState` is `virtual`."
+            },
+            "hypervisor": {
+              "$ref": "#/definitions/HypervisorConfiguration",
+              "description": "Hypervisor-related configuration. Present only when 'virtualizationState' is `hypervisor`."
+            }
+          }
+        }
+      }
+    },
+    "Process": {
+      "description": "A process resource represents a process running on a machine. The process may be actively *monitored*, i.e., a Dependency Agent is running on its machine, or *discovered*, i.e., its existence was inferred by observing the data stream from monitored machines. A process resource represents a pool of actual operating system resources that share command lines and metadata. As the process pool evolves over time, prior versions of the process resource are preserved and available for access. A process is live during an interval of time, if that process is executing during (parts) of that interval",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CoreResource"
+        }
+      ],
+      "x-ms-discriminator-value": "process",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "description": "Resource properties.",
+          "properties": {
+            "timestamp": {
+              "type": "string",
+              "format": "date-time",
+              "description": "UTC date and time when this process resource was updated in the system"
+            },
+            "monitoringState": {
+              "$ref": "#/definitions/MonitoringState",
+              "description": "Specifies whether the process is actively monitored or discovered."
+            },
+            "machine": {
+              "$ref": "#/definitions/ResourceReference",
+              "description": "Machine hosting this process."
+            },
+            "executableName": {
+              "type": "string",
+              "description": "The name of the process executable"
+            },
+            "displayName": {
+              "type": "string",
+              "description": "Name to use for display purposes"
+            },
+            "startTime": {
+              "type": "string",
+              "format": "date-time",
+              "description": "UTC date and time when the process started"
+            },
+            "role": {
+              "type": "string",
+              "enum": [
+                "webServer",
+                "appServer",
+                "databaseServer",
+                "ldapServer",
+                "smbServer"
+              ],
+              "description": "The inferred role of this process based on its name, command line, etc."
+            },
```

Comment:
This is the only part of the file that changed.  We added the "role" section.  I don't understand why GitHub seems to think that this entire file is new.